pkg/apis/core/validation/validation.go

```diff
@@ -1815,8 +1814,6 @@ func ValidatePersistentVolumeClaimUpdate(newPvc, oldPvc *core.PersistentVolumeCl
 	if utilfeature.DefaultFeatureGate.Enabled(features.BlockVolume) {
 		allErrs = append(allErrs, ValidateImmutableField(newPvc.Spec.VolumeMode, oldPvc.Spec.VolumeMode, field.NewPath("volumeMode"))...)
 	}
-
```

Comment:
Is this already being done in strategy prepareforupdate?

Reply:
yes, that is correct - https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/persistentvolumeclaim/strategy.go#L72